packages/iris-grid/src/IrisGridUtils.test.ts

```diff
@@ -658,8 +658,8 @@ describe('dehydration methods', () => {
       'dehydrateIrisGridPanelState',
```

Comment:
These tests right now are lacking hydration tests. We should add hydration tests for both v1 and v2 cases, to make sure we're getting the expected results. Some other cases to cover:
- No column selected (v1 and v2)
- One column selected (v1 and v2)
- Partial selection (v2 only)
- Multiple selected (v2 only)
... and any other you can think of.